modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= S3 - Compatible backup storage providers
+
+
+OADP supports a variety of object storage providers for different backup and snapshot operations.
+
+== S3 drivers for Backup Storage Locations
+
+=== Supported
+
+The following S3 compatible object storage providers, are fully supported by OADP through AWS plugin, for use as Backup Storage Locations:
+
+* Minio
+* MCG / nooba
+* AWS S3
+
+=== Unsupported
+
+The following S3 compatible object storage providers, are known to work with OADP through AWS plugin, for use as Backup Storage Locations, but are unsupported and have not been tested by Red Hat:
```

Comment:
```suggestion
The following S3 compatible object storage providers, are known to work with Velero through AWS plugin, for use as Backup Storage Locations, but are unsupported and have not been tested by Red Hat:
```

If it's not tested by Red Hat, I would suggest that they may work with Velero and not OADP directly.